.seed-config.yaml

```diff
@@ -0,0 +1,108 @@
+project:
+  name: "Godocit"
+  state: "unstable"
+  image: "/blob/master/logo.svg"
+  website: "https://godocit.okkur.org"
+  oneLiner: "Create documentation needed issues across different repositories"
+  description: |
+    When a PR introduces a new feature that requires documentation, this action
+    can automatically create an issue on your website's repository to add
+    documentation for that new feature.
+  version: "v0.1.0"
+  mainLicense: "apache2"
+  docsLicense: "CCBySa"
+
+vision:
+  type: ""
+  overview: |
+    Provide an automatic flow for keeping track of documentation.
+  items:
+    - 'Works easily by just adding a "documentation needed" label on the PR'
+    - "Create issues on different repositories"
+  concept: "Godocit's core concept is to provide a automatic tool to never miss
+    documentation for new features."
+  aim: "Godocit provides an easy and fast way to create documentation issues for
+    new features and keep track of undocumented features"
+
+readme:
+  usageExample: |
+    For using godocit you first need to create a Github App and install it on your
+    user or organization that has read and write access to the source and target
+    repositories. For more information read the ["Authenticating with GitHub Apps"](https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/) page.
```

Comment:
Might need to mention the PrivateKey being inserted into secrets instead of being used by a server component. At least that was the first thing that I didn't grasp right away as it turns the usual oauth flow a bit around.